ibis-server/tools/query_local_run.py

```
@@ -57,14 +58,22 @@
 with open(connection_info_path) as file:
     connection_info = json.load(file)
 
+# Extract the requried tables from the SQL query
+extractor = wren_core.ManifestExtractor(encoded_str)
+tables = extractor.resolve_used_table_names(sql)
+print("# Tables used in the SQL query:", tables)
```

Ruff (0.8.2)

`print` found

(T201)